webpack.config.js

```diff
@@ -6,21 +6,21 @@ module.exports = {
   mode: 'development',
   devServer: {
     static: './dist',
-    watchFiles:['src/index.html/index.js'] 
-  },
-  output: {
-    filename: 'main.js',
-    path: path.resolve(__dirname, 'dist'),
+    watchFiles: ['src/index.html/index.js'],
   },
+
   plugins: [
     new HtmlWebpackPlugin({
       template: './src/index.html',
     }),
   ],
+
```

Comment:
- It seems like you have not properly installed webpack in your project, I get this error when I try to run your project, please fix this 🙏 
![wg](https://user-images.githubusercontent.com/73607512/212665522-f042489c-27b4-4739-9deb-bb74d0b200ec.PNG)